crypto/rsa/rsa_err.c

```diff
@@ -118,6 +118,8 @@ static const ERR_STRING_DATA RSA_str_reasons[] = {
     "pub exponent out of range"},
     {ERR_PACK(ERR_LIB_RSA, 0, RSA_R_P_NOT_PRIME), "p not prime"},
     {ERR_PACK(ERR_LIB_RSA, 0, RSA_R_Q_NOT_PRIME), "q not prime"},
+    {ERR_PACK(ERR_LIB_RSA, 0, RSA_R_RANDOMNESS_SOURCE_STRENGTH_INSUFFICIENT),
+    "randomness source strength insufficient for keygen"},
```

Comment:
not sure if the `for keygen` should be also dropped from the text

Reply:
Dropped